src/lasso/issues/argparse.py

```diff
@@ -7,8 +7,6 @@
 
 def add_standard_arguments(parser: argparse.ArgumentParser):
     """Add normally expected command-line arguments to the given ``parser``."""
-    # The "version" option
-    parser.add_argument("--version", action="version", version=VERSION)
```

Comment:
@tloubrieu-jpl ☝️

Reply:
Thanks @nutjob4life , I removed it because it was colliding with my --version option and I did not understand what it was used for here. I am ok to put it back and rename the version argument I am using or re-use this one. Which version or the version of what is expected here ?

Reply:
When I run it right now, I get
```
$ milestones --version
1.1.0
$ pds-issues --version
1.1.0
$ pds-labels --version
1.1.0
```
However, if you delete lines 10–11 in `src/lasso/issues/argparse.py`, suddenly I get:
```
$ milestones --version
usage: milestones [-h] [--version] [-d | -q] [--github-org GITHUB_ORG] [--github-repos [GITHUB_REPOS ...]]
                  [--length LENGTH] [--token TOKEN] [--create] [--delete] [--close] [--due-date DUE_DATE]
                  [--sprint-name-file SPRINT_NAME_FILE] [--sprint-names [SPRINT_NAMES ...]]
                  [--prepend-number PREPEND_NUMBER]
milestones: error: unrecognized arguments: --version
$ pds-issues --version
usage: pds-issues [-h] [--version] [-d | -q] [--github-org GITHUB_ORG] [--github-repos [GITHUB_REPOS ...]]
                  [--token TOKEN] [--issue_state {open,closed,all}] [--start-time START_TIME] [--end-time END_TIME]
                  [--format FORMAT] [--build BUILD] [--loglevel {DEBUG,INFO,WARN,ERROR,CRITICAL}]
pds-issues: error: unrecognized arguments: --version
$ pds-labels --version
usage: pds-labels [-h] [--version] [-d | -q] [--github_org GITHUB_ORG] [--github_repos [GITHUB_REPOS ...]]
                  [--token TOKEN] [--label-name LABEL_NAME] [--label-color LABEL_COLOR] [--create] [--delete]
                  [--config_file CONFIG_FILE]
pds-labels: error: unrecognized arguments: --version
```
Maybe you could call yours `--stable-version`?

Reply:
Yes I will reverse that change and find a different name for my version, sorry I broke that.

Reply:
Hi @nutjob4life , I made that change. The --version work as it used to now.

Reply:
@tloubrieu-jpl no apologies needed!